readme.md

```diff
@@ -443,6 +443,10 @@ done manually or using OpenTracing framework instrumentations such as
 Framework instrumentation automatically propagates tracing headers and also creates in-process
 spans to better understand what is happening inside the application.
 
+There are different ways how the tracer can be configured. Customer Java service is using `tracerresolver`
```

Comment:
Slight rewrite: "There are different ways to configure the tracer. The Customer...."